### PR TITLE
Introduce Azure Maps

### PR DESCRIPTION
**Pull request by jp94**
## Description

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

---
Refer to: Azure/azure-sdk-for-net-pr#834
REST spec PR: Azure/azure-rest-api-specs-pr#440

### CODE REVIEW

File: src/SDKs/Maps/Management.Maps/Microsoft.Azure.Management.Maps.csproj [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.Maps</PackageId>
+    <PackageReleaseNotes>This package replaces Microsoft.Azure.Management.LocationBasedServices. Azure Location Based Services is now Azure Maps.</PackageReleaseNotes>
```
Comment (jp94) [inline src/SDKs/Maps/Management.Maps/Microsoft.Azure.Management.Maps.csproj]: "Open a PR for the Microsoft.Azure.Management.Maps csproj, indicate that this is a replacement for the Microsoft.Azure.Management.LocationBasedServices in the PackageReleaseNotes and release the package"

File: src/SDKs/Maps/Management.Maps/Microsoft.Azure.Management.Maps.csproj [hunk 2]
```diff
@@ -0,0 +1,14 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
```
Comment (dsgouda) [inline src/SDKs/Maps/Management.Maps/Microsoft.Azure.Management.Maps.csproj]: Copy lines 2-4 at the top and lines 22-24 at the bottom from [here](https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/Compute/Management.Compute/Microsoft.Azure.Management.Compute.csproj)

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, please address the comment and run
`msbuild build.proj /t:build /p:Scope=SDKs\Maps` and commit any changes

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**